audioop/_audioop.c.h

```diff
@@ -68,8 +68,13 @@ _LTS_PyArg_BadArgument(const char *fname, const char *displayname,
     const char *arg_type_name_chars = PyUnicode_AsUTF8AndSize(arg_type_name, NULL);
     if (arg_type_name_chars == NULL) {
         Py_DECREF(arg_type_name);
-        Py_DECREF(arg_type);
-        return;
+        arg_type_name = PyType_GetQualName(arg_type);
```

Comment:
I was tempted to use ``SETREF`` here but held off just in case it was not the correct thing to do.